src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -64,14 +69,27 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             startingDirectory = _startingDirectory;
         }
 
+        EnvironmentVariableMapW extraEnvVars;
+        {
+            // Convert connection Guid to string and ignore the enclosing '{}'.
+            std::wstring wsGuid{ Utils::GuidToString(_guid) };
+            wsGuid.pop_back();
+
+            WCHAR* pwszGuid{ wsGuid.data() + 1 };

```

Comment:
can the type here be `const wchar_t* const`?

Reply:
Sure.  Does this project generally prefer the native types over the Windows ones?

Reply:
I'm not sure we have a consensus on that, I think we typically switch to native types when we refactor an older piece of code.